.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "sde.languageServerMode": "sourcekit-lsp"
+}
```

Comment:
Should this be checked in? Looks like it's related to an [extension](https://marketplace.visualstudio.com/items?itemName=vknabel.vscode-swift-development-environment)